src/test/TemplateProcessor.test.js

```diff
@@ -1229,6 +1229,14 @@ test("test rxLog", async () => {
 
 });
 
+test("test onInit", async () => {
+    const tp = new TemplateProcessor({});
+    let beenCalled = false;
+    TemplateProcessor.onInit = () => { beenCalled = true;}
```

Comment:
Per earlier comment, this sets onInit for *all* template processors. Instead, do `tp.onInit  = () => { beenCalled = true;}` In this way you will set onInit for just this one template processor, not all of them.

Reply:
changed it as well.